REVIEW_GUIDELINES.md

```diff
@@ -112,32 +112,51 @@ volumes:
   {{- end -}}
 ```
 
-* Example pvc.yaml
+## AutoScaling / HorizontalPodAutoscaler
```

Comment:
@paulczar The autoscaling addition looks good 👍 But did you mean to delete the example PVC YAML?

Reply:
lulz wtf how did I manage that.  I can't even copy and paste good!